java/threading/multithreading/why-use-volatile-long-or-double.md

```diff
@@ -57,4 +57,8 @@ In order to avoid this problem, `long` and `double` values need to be granted at
 private volatile long i = 0;
 ```
 
-Making the field `volatile` will guarantee its atomicity.
\ No newline at end of file
+We've previously said that making the field `volatile` **will not guarantee its atomicity**. This is true, especially in the case of operations that require more than one read or write.
+
+Making a variable `volatile` will guarantee that **each individual read or write operation** on the variable will be atomic.
```

Comment:
These additions look good 👍 

But I do have one concern, does the reader know what "atomic" means in this context? I can't find a clear explanation of it anywhere in the insight.

It's not a commonly used word in regular language so people new to programming (and especially non native speakers) might be confused by it.

Shall we add a footnote explaining it? Something like

> In programming, an operation (or a set of operations) is considered atomic if it appears to the rest of the system as a single, indivisible step.

Reply:
yeah, we certainly should. I've also re-ordered the insight a bit to say that this does not only apply to `long` or `double`